README.md

```diff
@@ -1,183 +1,20 @@
-
-```sh
-git clone --depth 1 https://github.com/developit/preact-boilerplate.git my-app
-cd my-app
-```
-
-
-**2. Make it your own:**
-
-```sh
-rm -rf .git && git init && npm init
-```
-
-> :information_source: This re-initializes the repo and sets up your NPM project.
-
-
-**3. Install the dependencies:**
+## Setup
 
 ```sh
 npm install
-```
-
-> You're done installing! Now let's get started developing.
-
-
-
-## Development Workflow
-
-
-**4. Start a live-reload development server:**
-
-```sh
 npm run dev
 ```
 
-> This is a full web server nicely suited to your project. Any time you make changes within the `src` directory, it will rebuild and even refresh your browser.
-
-**5. Testing with `mocha`, `karma`, `chai`, `sinon` via `phantomjs`:**
-
-```sh
-npm test
-```
-
-> 🌟 This also instruments the code in `src/` using [isparta](https://github.com/douglasduteil/isparta), giving you pretty code coverage statistics at the end of your tests! If you want to see detailed coverage information, a full HTML report is placed into `coverage/`.
-
-**6. Generate a production build in `./build`:**
+## Build
 
 ```sh
 npm run build
 ```
 
-> You can now deploy the contents of the `build` directory to production!
->
-> **[Surge.sh](https://surge.sh) Example:** `surge ./build -d my-app.surge.sh`
-> 
-> **[Netlify](https://www.netlify.com/docs/cli/) Example:** `netlify deploy`
->
-> [![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/developit/preact-boilerplate)
-
-
-**5. Start local production server with [serve](https://github.com/zeit/serve):**
-
-```sh
-npm start
-```
-
-> This is to simulate a production (CDN) server with gzip. It just serves up the contents of `./build`.
-
-
-
----
-
-
-## Structure
-
-Apps are built up from simple units of functionality called Components. A Component is responsible for rendering a small part of an application, given some input data called `props`, generally passed in as attributes in JSX. A component can be as simple as:
-
-```js
-class Link extends Component {
-  render({ to, children }) {
-    return <a href={ to }>{ children }</a>;
-  }
-}
-// usage:
-<Link to="/">Home</Link>
-```
-
-
----
-
-
-## CSS Modules
-
-This project is set up to support [CSS Modules](https://github.com/css-modules/css-modules).  By default, styles in `src/style` are **global** (not using CSS Modules) to make global declarations, imports and helpers easy to declare.  Styles in `src/components` are loaded as CSS Modules via [Webpack's css-loader](https://github.com/webpack/css-loader#css-modules).  Modular CSS namespaces class names, and when imported into JavaScript returns a mapping of canonical (unmodified) CSS classes to their local (namespaced/suffixed) counterparts.
-
-When imported, this LESS/CSS:
-
-```css
-.redText { color:red; }
-.blueText { color:blue; }
-```
-
-... returns the following map:
-
-```js
-import styles from './style.css";
```

Comment:
I just nuked the readme to have it for Traitify purposes. I think we could link to the original in a contributing section if you want but the I think the bulk of it should be about Traitify.